storybook/config.js

```diff
@@ -6,11 +6,132 @@ import i18n from '@/i18n'
 
 import vocabularyTheme from './theme'
 
+// Configuration of addon-viewport
+
+const newViewports = {
```

Comment:
You should put these in a separate file like `viewport.js` and import. Refer to theme from line 7.